README.md

```diff
@@ -1 +1,99 @@
 # deis-cleanup
+It's [Deis](https://github.com/deis/deis/) specific approach to `docker image`
+cleanup. Currently available solutions are too io heavy or aren't configurable
+enough.
+
+## Usage
+### Environment variables
+Script can be configured by setting environement variables. It can by passed to
+docker run (`docker run --env "ENV_NAME=value" ...`).
+
+- `KEEP_LAST_VERSIONS` - the number of releases keep for each repository (default: `2`).
```

Comment:
https://github.com/Ragnarson/deis-cleanup/pull/1#discussion_r55812935

releases **to** keep